lisa/operating_system.py

```diff
@@ -797,13 +805,23 @@ def add_repository(
         if type(self._node.os) == Debian:
             self._node.execute("apt-get update", sudo=True)
 
+    def is_end_of_life_release(self) -> bool:
+        for release in self.end_of_life_releases:
+            if self.information.full_version == release:
+                return True
+        return False
```

Comment:
return self.information.full_version in self.end_of_life_releases

Reply:
Done